README.md

```diff
@@ -1,3 +1,6 @@
+> **Warning**
+> This project is archived. The Dataset Browser is moved to the [Colonial Collections monorepo](https://github.com/colonial-heritage/dataset-browser).
```

Comment:
Change the URL to https://github.com/colonial-heritage/colonial-collections instead?